src/DacpacTool/PackageBuilder.cs

```diff
@@ -388,5 +388,13 @@ private List<int> ParseSuppressionList(string suppressionList)
 
             return result;
         }
+
+        public void GenerateCreateScript(FileInfo dacpacFile, string databaseName)
+        {
+            databaseName ??= Path.GetFileNameWithoutExtension(dacpacFile.Name);
+            using var package = DacPackage.Load(dacpacFile.FullName);
+            using var file = File.Create($"{databaseName}_Create.sql");
+            DacServices.GenerateCreateScript(file, package, databaseName);
```

Comment:
Seems simple enough! Would you please add some tests that validate this in action?